src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx

```diff
@@ -102,25 +105,90 @@ const TeamMembers = createReactClass({
         });
       },
     });
+
+    this.api.request(`/organizations/${params.orgId}/members/`, {
+      success: data => {
+        this.setState({
+          orgMemberList: data,
+        });
+      },
+      error: () => {
+        IndicatorStore.add(t('Unable to load organization members.'), 'error', {
+          duration: 2000,
+        });
+      },
+    });
   },
 
-  addMemberButton(access, orgId) {
-    return access.has('org:write') ? (
-      <Button
-        priority="primary"
-        size="small"
-        className="pull-right"
-        to={`/settings/organization/${orgId}/members/new/`}
-      >
-        <span className="icon-plus" /> {t('Invite Member')}
-      </Button>
-    ) : (
-      <a
-        className="btn btn-primary btn-sm btn-disabled tip pull-right"
-        title={t('You do not have enough permission to add new members')}
-      >
-        <span className="icon-plus" /> {t('Invite Member')}
-      </a>
+  addTeamMember(selection) {
+    let params = this.props.params;
+
+    this.setState({
+      loading: true,
+    });
+
+    joinTeam(
```

Comment:
i think ideally this would live in an action creator, but maybe not necessary in this pr

Reply:
i thought `joinTeam` was from an action creator (`import {joinTeam, leaveTeam} from '../../../actionCreators/teams'`). or do you mean something else?